guides/release/models/defining-models.md

```diff
@@ -51,15 +52,16 @@ properties that combine or transform primitive attributes.
 ```javascript {data-filename=app/models/person.js}
 import DS from 'ember-data';
 import { computed } from '@ember/object';
+const { Model, attr } = DS;
 
-export default DS.Model.extend({
-  firstName: DS.attr(),
-  lastName: DS.attr(),
+export default class Person extends Model {
+  @attr() firstName;
+  @attr() lastName;
 
   fullName: computed('firstName', 'lastName', function() {
```

Comment:
This mix of syntaxes has problems. See #554.

Reply:
Yup, I think for now we can just make this a standard ES6 getter, and not a computed property